project/akeneo.py

```diff
@@ -0,0 +1,11 @@
+from .remote import RemoteProject
+
+
+class Akeneo(RemoteProject):
+    major_version = 'v1'
+    remote = 'https://github.com/akeneo/pim-community-standard.git'
+
+    @property
+    def platformify(self):
+        return super(Akeneo, self).platformify + [
+                'cd {0} && composer update --no-scripts && composer require platformsh/config-reader --no-scripts'.format(self.builddir),]
```

Comment:
I don't think we need the composer update on this line; that should happen twice already elsewhere.

Reply:
composer update has been removed